02/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const liElements = document.querySelectorAll('a[data-url');
+
+if (liElements) {
```

Comment:
W przypadku wyszukiwania wielu elementów nie ma sensu to sprawdzenie bo tablica (czy pusta czy nie) i tak jest prawdą bo to obiekt. Jeśli już chcesz dać warunek to trzeba zrobić tak: `liElements.length > 0`